### PR TITLE
refactor: remove nunjucks

### PR DESCRIPTION
**Pull request by dead-horse**
1. remove nunjucks and options.template support.
2. add ctx as the second argument to custom handlers to be friendly with arrow functions.

### CODE REVIEW

**Comment by fengmk2** [comment]
should rebase from master

**Comment by fengmk2** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -62,7 +62,6 @@ onerror(app, options);
 * **text**: text error handler
 * **json**: json error handler
 * **html**: html error handler
-* **template**: default html error handler template path
```
Comment (fengmk2) [inline README.md]: major change

Comment (dead-horse) [inline README.md]: en, must release 3.0.0

**Comment by fengmk2** [comment]
rebase before merge

**Comment by atian25** [comment]
ping

**Comment by dead-horse** [comment]
done

**Comment by fengmk2** [comment]
+1

**Pull request merged**